OWNERS

```diff
@@ -48,7 +48,7 @@
     },
     {
       // Locked down to minimize new import aliases being added
-      pattern: 'tsconfig{,.base}.json',
+      pattern: '**/tsconfig{,.base}.json',
```

Comment:
This means all tsconfig files in all directories require wg-performance approval. May I suggest splitting this into two rules:
1. root `tsconfig{,.base}.json` with `required: true`
2. recursive `**/tsconfig.json` without (also doesn't need the base bit)

Reply:
I'm actually very unfamiliar with how our OWNERS files work. My primary goal is to ensure me & you are able to modify any tsconfig file without needing approval from other working groups. My impression is that we get to a bit lazier than most about it due to working closely with Justin

Reply:
Read up on the docs, LGTM. Making the update

Reply:
Have you seen https://ampproject-owners-bot.appspot.com/example?

Reply:
That was the doc I read. Although not the nice HTML version of it, the one in ampthml